ImplementationGuide/markdown/Motivation.md

```diff
@@ -1,12 +1,12 @@
 # Motivation
 
-Innerhalb des [ISiK Basismoduls](https://simplifier.net/guide/implementierungsleitfadenisik-basismodul-stufe-3/ImplementationGuide-markdown-Einfuehrung?version=current) erfolgt eine Erörterung der [Motivation für den Einsatz einer ISiK-konformen Schnittstelle](https://simplifier.net/guide/implementierungsleitfadenisik-basismodul-stufe-3/ImplementationGuide-markdown-Motivation?version=current). Ein entsprechender Fokus liegt hierbei auf der Integration von spezialisierten Subsystemen, welche entweder Informationen am Krankenhausinformationssystem (KIS) oder Klinischen Arbeitsplatzsystem (KAS) abrufen oder Informationen an die zuvor genannten Systeme zurückliefern. Hervorgehoben sei hier die Integration von mobilen Anwendungen und Web-Portalen. 
+Innerhalb des [ISiK Basismoduls](https://simplifier.net/guide/basis-v3?version=current) erfolgt eine Erörterung der [Motivation für den Einsatz einer ISiK-konformen Schnittstelle](https://simplifier.net/guide/basis-v3/ImplementationGuide-markdown-Motivation?version=current). Ein entsprechender Fokus liegt hierbei auf der Integration von spezialisierten Subsystemen, welche entweder Informationen am Krankenhausinformationssystem (KIS) oder Klinischen Arbeitsplatzsystem (KAS) abrufen oder Informationen an die zuvor genannten Systeme zurückliefern. Hervorgehoben sei hier die Integration von mobilen Anwendungen und Web-Portalen. 
```

Comment:
issue (blocking): dieser einführende Absatz ist unklar.

Es wird auf das Basismodul verwiesen. Die Verwendung von "hierbei" und "hier" sorgt für Unklarheit, was gemeint ist. 
Grundsätzlicher gilt jedoch:

Reply:
todo: dieser Abschnitt und die gesamte Seite müssen überarbeitet werden.

Es fehlen:
- Die Identifikation und Feststellung von **Bedarfen** rund um die Client-Anbindung, insbesondere fehlt eine
**Beschreibung von Use Cases (auf Versorgungsebene - oder zumindest IT-Workflow/-Projekt-Ebene)** wofür die Client-Integration nützlich sein soll.
- Bedarfe und Use Cases sollen als solche kenntlich gemacht werden
- Es sollten klare **Ziele** formuliert werden, à la: 
    - "Durch die Implementierung von ISIK-Connect kann ein Krankenhaus (vereinfacht) Services über die KH-IT anbieten, wie z.B. Erhebung von Vitalparametern durch Smart-Devices." 
    - oder : "Durch ISIK-Connect kann die KH-IT nicht-proprietäre Lösungen in das eigene Portfolio einführen."
    -  oder: "Durch ISIK-Connect kann die KH-IT ihre Integrationsprojekte vereinfachen, indem sie ..."
    - oder " Durch die Infrastruktur ISIK-Connect soll ein Patient befähigt über seine Smart-Watch Vitalparameter Daten an ein Krankenhaus zu übermitteln (bei dem er in Behandlung ist)."

Reply:
suggestion: ergänzend, sollten wir den Ist-Stand und die aktuellen Herausforderungen aufzeigen:
- Wie erfolgt die Client-Integration in die IT-Landschaft jetzt, also ohne SMART-on-FHIR? 
- Was sind die heutigen Probleme der Client-Integration? Wer ist von den Problemen direkt / indirekt betroffen, welche Nachteile entstehen dabei?
- Wie würde SMART-on-FHIR die aufgezeigten Probleme lösen? Welche Vorteile ergeben sich dabei für die Stakeholder?

question: meiner Wahrnehmung nach versucht die Spec den Markt für "Apps" im Krankenhausumfeld zu schaffen und für sie einen standardisierten Autorisierungsmechanismus auf (KIS-)Daten zu definieren. Ist es richtig? Sind weitere Sicherheit-relevante Probleme in Krankenhaus-IT im Fokus, z.B.:
- Zentralisierte Verwaltung von Zugriffsberechtigungen
- Authentifizierung / Autorisierung von Backend-Systemen untereinander (KIS <-> Archivsystem, PDMS <-> Labor, KIS <-> Patientenportal Backend etc.)?